index.html

```diff
@@ -30,7 +30,36 @@
     <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
     <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
     <![endif]-->
+    <!-- For Lightbox Effect-->
+    <link rel="stylesheet" type="text/css" media="screen"
+          href="http://cdnjs.cloudflare.com/ajax/libs/fancybox/1.3.4/jquery.fancybox-1.3.4.css"/>
+    <link rel="stylesheet" type="text/css" media="screen"
+          href="http://cdnjs.cloudflare.com/ajax/libs/fancybox/1.3.4/jquery.fancybox-1.3.4.css"/>
+    <style type="text/css">
```

Comment:
if possible, please move this code to a css file instead of it being written inside the html code